rpcs3/rpcs3qt/game_list_frame.cpp

```diff
@@ -449,6 +449,11 @@ void game_list_frame::Refresh(const bool from_drive, const bool scroll_after)
 		m_notes.clear();
 		m_games.pop_all();
 
+		if (Emu.IsStopped())
```

Comment:
why is this suddenly a criteria?

Reply:
Here it's so the string won't construct when stopped, and in inner code BootGame behaves wrongly with it unstopped. It was a bug I believe.

Reply:
you're not supposed to add games to the games.yml list in the refresh method anyway.

Reply:
but i guess it doesn't hurt